Formula/boost-python3.rb

```diff
@@ -6,15 +6,8 @@ class BoostPython3 < Formula
   head "https://github.com/boostorg/boost.git", branch: "master"
 
   stable do
```

Comment:
Remove the `revision 1` above these lines. Also should get rid of `stable do` block and move `url`/`sha256` after the `homepage`.